### ISSUE TITLE
Toggle handleSelect is triggered upon rendering

### ISSUE THREAD

**Issue by StefBrands** [tremorlabs/tremor-npm#152]
I noticed that the handleSelect in the Toggle component is triggered when the component is rendered. I've got the following Toggle inside a ReactModal and as soon as the modal is opened the handleSelect triggers, causing unwanted API requests. I'm not completely sure if I'm making a mistake somewhere so I'm wondering if more people have noticed this behaviour.

``` 
<Toggle defaultValue={"2"} color="emerald" handleSelect={(value) => postMood(value)}>
                    <ToggleItem value={"1"} text="Great"/>
                    <ToggleItem value={"2"} text="Okay"/>
                    <ToggleItem value={"3"} text="Bad"/>
</Toggle>
```

**Comment by mitrotasios** [comment]
Hi @StefBrands, do you mind sharing the complete code snippet?

**Comment by StefBrands** [comment]
> 

Of course:
```
async function postMood(mood) {
        console.log(mood)
        more api stuff here
        }
```
Logging as soon as it enters `postMood()` shows output when the modal gets opened.

**Comment by mitrotasios** [comment]
hm yeah you're right, that shouldn't be the case. Will try to fix this as soon as possible. Thanks for pointing it out!

**Comment by mitrotasios** [comment]
hey @StefBrands, the issue should be fixed with the new release [v1.0.12](https://github.com/tremorlabs/tremor/releases/tag/v1.0.12).

**Comment by StefBrands** [comment]
> hey @StefBrands, the issue should be fixed with the new release [v1.0.12](https://github.com/tremorlabs/tremor/releases/tag/v1.0.12).

Working great now @mitrotasios! Great job, super fast

**Comment by mitrotasios** [comment]
perfect, thanks for using our library and being so involved :)
